arcade/physics_engines.py

```diff
@@ -432,17 +450,51 @@ def disable_multi_jump(self):
         self.allowed_jumps = 1
         self.jumps_since_ground = 0
 
-    def jump(self, velocity: int):
-        """ Have the character jump. """
-        self.player_sprite.change_y = velocity
-        self.increment_jump_counter()
+    def jump(self, velocity: int,
+             air_jump_velocity: Optional[int] = None,
+             air_jump_style: Optional[str] = "set",
+             jump_velocity_limit: Optional[int] = None):
+        """ Have the character jump. Multijump can be set with a separate in-air velocity and air jumps can be
+        set to be additive, limited, or a set value. Additive only adds to the player's change_y velocity. Limited
+        will add to the players' change_y until the jump_velocity limit. Set always sets the players velocity
+        to their air jump speed. """
+        if self.can_jump():
+            # Air Jump logic
+            if self.jumps_since_ground > 0:
+                # This checks if air_jump_velocity is set. If not it will default to the velocity for all air jumps.
+                if air_jump_velocity:
+                    air_jump = air_jump_velocity
+                else:
+                    air_jump = velocity
+                if air_jump_style == "additive":
```

Comment:
Would it be better to use a constant/enum for this?>

Reply:
Looking into it.

Reply:
Does the current version of the code better fulfill your intent?